### PR TITLE
[6.14.z] Mark UI tests as end-to-end for Rocket Team Components

### PR DESCRIPTION
**Pull request by shweta83**
### Problem Statement
Manual cherrypick of https://github.com/SatelliteQE/robottelo/issues/14997

### CODE REVIEW

**Comment by shweta83** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui --team Rocket -m e2e

File: tests/foreman/cli/test_leapp_client.py [hunk 1]
```diff
@@ -295,86 +295,3 @@ def test_leapp_upgrade_rhel(
...
-        {'source_version': RHEL8_VER, 'target_version': RHEL9_VER},
-    ],
-    ids=lambda upgrade_path: f'{upgrade_path["source_version"]}'
-    f'_to_{upgrade_path["target_version"]}',
-)
-def test_leapp_upgrade_rhel_non_admin(
-    module_target_sat,
-    module_sca_manifest_org,
-    default_location,
-    custom_leapp_host,
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_leapp_client.py:312]: we recently added this test, why removing this now for 6.14.z?

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 7309
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/ui --team Rocket -m e2e --external-logging
Test Result : = 4 failed, 23 passed, 1 skipped, 642 deselected, 1569 warnings in 14850.17s (4:07:30) =
```

File: tests/foreman/cli/test_leapp_client.py (hunk 1 above)
Comment (shweta83) [reply tests/foreman/cli/test_leapp_client.py]: It was removed by mistake.

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**